lib/cli/src/Cardano/CLI.hs

```diff
@@ -1146,15 +1155,35 @@ verbosityToMinSeverity = \case
 
 -- | Initialize logging at the specified minimum 'Severity' level.
 initTracer
-    :: Severity
-    -> Text
+    :: Maybe FilePath
+    -> Severity
     -> IO (CM.Configuration, Switchboard Text, Trace IO Text)
-initTracer minSeverity cmd = do
-    c <- defaultConfigStdout
-    CM.setMinSeverity c minSeverity
-    CM.setSetupBackends c [CM.KatipBK, CM.AggregationBK]
-    (tr, sb) <- setupTrace_ c "cardano-wallet"
-    pure (c, sb, appendName cmd tr)
+initTracer configFile minSeverity = do
+    let defaultConfig = do
+            c <- defaultConfigStdout
+            CM.setMinSeverity c minSeverity
+            CM.setSetupBackends c [CM.KatipBK, CM.AggregationBK]
+            pure c
+    cfg <- maybe defaultConfig CM.setup configFile
+    (tr, sb) <- setupTrace_ cfg "cardano-wallet"
+    pure (cfg, sb, tr)
+
+-- | Run an action with logging available and configured. When the action is
+-- finished (normally or otherwise), log messages are flushed.
+withLobemo
```

Comment:
do not understand why it is called `withLobemo` ;-)

Reply:
Me neither, I was wondering the same.

Reply:
Logging, benchmarking, monitoring.